### ISSUE TITLE
Advanced settings: error when activating the "restore default settings" button

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#11341]
### Steps to reproduce:
1. Open NVDA's avanced settings
2. Press restore default settings

### Actual behavior:
Following error:
```
IO - inputCore.logTimeSinceInput (21:47:14.775) - MainThread (2824):
0.039 sec since input
IO - speech.speak (21:47:14.775) - MainThread (2824):
Speaking ['Standard-Einstellungen wiederherstellen', 'Schalter']
IO - inputCore.InputManager.executeGesture (21:47:15.586) - winInputHook (15028):
Input: kb(laptop):space
IO - inputCore.logTimeSinceInput (21:47:15.596) - MainThread (2824):
0.010 sec since input
IO - speech.speak (21:47:15.596) - MainThread (2824):
Speaking ['gedrückt']
ERROR - unhandled exception (21:47:15.696) - MainThread (2824):
Traceback (most recent call last):
  File "gui\settingsDialogs.pyc", line 2574, in <lambda>
  File "gui\settingsDialogs.pyc", line 2500, in restoreToDefaults
AttributeError: 'Choice' object has no attribute 'SetValue'
```

### Expected behavior:
No error should occur

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
Version: alpha-20514,e87d8ab1

#### Windows version:
Windows 10 2004 update

#### Name and version of other software in use when reproducing the issue:
n/a

#### Other information about your system:
n/a

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no

#### If addons are disabled, is your problem still occuring?
yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
yes, no effect.

**Comment by XLTechie** [comment]
I can confirm this, although in Alpha 20491, the line numbers are lower by one 
in each case.

**Comment by Adriani90** [comment]
Closing as fixed by #11344
